packages/features/ee/workflows/pages/workflow.tsx

```diff
@@ -172,30 +177,47 @@ function WorkflowPage() {
       form={form}
       handleSubmit={async (values) => {
         let activeOnEventTypeIds: number[] = [];
+        let isEmpty = false;
 
         values.steps.forEach((step) => {
+          const isSMSAction =
+            step.action === WorkflowActions.SMS_ATTENDEE || step.action === WorkflowActions.SMS_NUMBER;
+          const isBodyEmpty =
+            step.template === WorkflowTemplates.CUSTOM &&
+            !isSMSAction &&
+            convert(step.reminderBody).length <= 1;
+          if (isBodyEmpty) {
```

Comment:
For custom email bodies it needs to be checked if it is not empty. For all other input fields, the browser form validation is used